app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -7,16 +7,12 @@ class PermissionTemplateForm
       self.model_class = PermissionTemplate
       self.terms = []
       delegate :access_grants, :access_grants_attributes=, :release_date, :release_period, :visibility, to: :model
-      delegate :available_workflows, :active_workflow, :source_model, to: :model
-
-      # @return [#to_s] the primary key of the associated admin_set or collection
-      # def source_id (because you might come looking for this method)
-      delegate :id, to: :source_model, prefix: :source
+      delegate :available_workflows, :active_workflow, :source, :source_id, to: :model
 
       ##
-      # @deprecated  use PermissionTemplate#reset_access_controls instead.
+      # @deprecated  use PermissionTemplate#reset_access_controls_for instead.
       def reset_access_controls!
-        Deprecation.warn("reset_access_controls! is deprecated; use PermissionTemplate#reset_access_controls instead.")
+        Deprecation.warn("reset_access_controls! is deprecated; use PermissionTemplate#reset_access_controls_for instead.")
```

Comment:
The long term method that should be used is `#reset_access_controls_for`, so the deprecation notice was updated.